### ISSUE TITLE
Verifying hash files doesn't work under Windows 7 64-bit.

### ISSUE THREAD

**Issue by nightson** [rhash/RHash#119]
**OS:** Windows 7 SP1 64-bit.

**RHash Version:**  1.3.9

```
# rhash.exe -C rhash.exe -o rhash.sfv
# rhash.exe -c rhash.sfv
--( Verifying rhash.sfv )-------------------------------------------------------
RHash: (stdout): No error
```

When verifying the sfv, Rhash didn't even check the file. Instead, it gave a no error message instantly. 

On the other hand, 1.3.8 works just fine.
```
# rhash1.3.8.exe -c rhash.sfv
--( Verifying rhash.sfv )-------------------------------------------------------
rhash.exe                                           OK
--------------------------------------------------------------------------------
Everything OK
```

Is this a bug?

**Comment by rhash** [comment]
Bug confirmed. Output to console in utf8 encoding is broken on Win7 SP1.

Workaround: specify --dos or --win option, or redirect program output to a file.

**Comment by rhash** [comment]
Fixed by 05038f8486381248896f20ff1e72903e3498ac6e.

Thanks for reporting the problem!

FYI: The culprit is the bug in the Windows 7 itself: Windows, after a `fwprintf()`/`vfwprintf()` call, incorrectly sets the _IOERR flag (causing `ferror() != 0`) for a console output stream in the CP_UTF8 codepage.

**Comment by rhash** [comment]
The fix is included into **RHash v1.4.0**.
